data-structures/data.md

```diff
@@ -0,0 +1,28 @@
+`Flexible Byte Layout` is an advanced layout for representing binary data.
+
+It is flexible enough to support very small and very large (multi-block) binary data.
+
+```sh
+type NestedByte struct {
+  lengthHint Int
```

Comment:
> What is a concise way to describe “unverified” or “insecure” information that we can attach to arbitrary nouns?

I am going to list a bunch of things back to back, **without** being happy with any of them: perhaps something will stand out...

```
length Int
lengthHint Int
unverifiedLength Int
insecureLength Int
uncertifiedLength Int
uncertLength Int
forgeableLength Int
untrustedLength Int
uncheckedLength Int
metaLength Int
unauditedLength Int
```